### PR TITLE
Add fixture for non-image algorithm

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**

### CODE REVIEW

File: scripts/algorithm_evaluation_fixtures.py [hunk 1]
```diff
@@ -53,6 +59,10 @@ def _get_inputs():
     )
 
 
+def _get_non_image_inputs():
+    return ComponentInterface.objects.filter(slug__in=["predictions-csv-file"])
+
```
Comment (amickan) [inline scripts/algorithm_evaluation_fixtures.py:64]: Don't we want a test with json type interface that is not stored in the DB as well?

**Review by amickan — APPROVED** [review]

**Comment by jmsmkn** [comment]
Why was this done? Is it used anywhere or only for convenience?

The suffix incrementing has made working with this command annoying - this is often run multiple times when developing the algorithm evaluation feature and you get several algorithms with the same title but different inputs, unclear which to select when submitting to the demo algorithm.

**Comment by MikeOverkamp-diag** [comment]
This was added to be able to test https://github.com/DIAGNijmegen/rse-gcapi/pull/137 properly.

**Comment by jmsmkn** [comment]
So is it used in the gcapi tests?

**Comment by MikeOverkamp-diag** [comment]
Yes.

**Pull request merged**